### ISSUE TITLE
Problem with underscores in RPC names 

### ISSUE THREAD

**Issue by tagantroy** [grpc/grpc-kotlin#106]
Hello, I upgraded to version 0.1.2 and realized that compilation still failing.
I added to examples/src/main/proto/hello_world.proto
```
+  rpc say_hello (HelloRequest) returns (HelloReply);
```
And I got
```
> Task :compileKotlin FAILED
e: /Users/ivanbalaksha/work/grpc-kotlin-upstream/examples/build/generated/source/proto/main/grpckt/io/grpc/examples/helloworld/HelloWorldGrpcKt.kt: (41, 25): Unresolved reference: getsay_helloMethod
e: /Users/ivanbalaksha/work/grpc-kotlin-upstream/examples/build/generated/source/proto/main/grpckt/io/grpc/examples/helloworld/HelloWorldGrpcKt.kt: (84, 19): Unresolved reference: getSay_helloMethod
e: /Users/ivanbalaksha/work/grpc-kotlin-upstream/examples/build/generated/source/proto/main/grpckt/io/grpc/examples/helloworld/HelloWorldGrpcKt.kt: (133, 32): Unresolved reference: getSay_helloMethod
```
Looks like wrong build was uploaded

**Comment by bshaffer** [comment]
Fixed in [0.1.2](https://github.com/grpc/grpc-kotlin/releases/tag/v0.1.2)!

**Comment by dariuszkuc** [comment]
Getting similar problem, given `hello_world.proto`

```
syntax = "proto3";

option java_multiple_files = false;
package hello;

service Hello {
    rpc hello(Request) returns (Response);
}

message Request {
    string name = 1;
}

message Response {
    string greeting = 1;
}
```

`grpc-kotlin` generates `HelloWorldGrpcKt.HelloGrpcKt` with

```kotlin
val helloMethod: MethodDescriptor<HelloWorld.Request, HelloWorld.Response>
  @JvmStatic
  get() = HelloGrpc.gethelloMethod()  // compilation error, actual method name is getHelloMethod()
```

**Comment by tagantroy** [comment]
@dariuszkuc I tried with latest master and it is works, you can test is by yourself. Change version in build.gradle and run `./gradlew publishToMavenLocal`, after that update version in your project/sample and try again. For me it works for:
```
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply);
  rpc hello_my_friend (HelloRequest) returns (HelloReply);
  rpc hello(HelloRequest) returns (HelloReply);
}
```
Looks like 0.1.2 version built without 045e1eedef3379db477aff0f0fa5a327c55ea8a9 commit

**Comment by bshaffer** [comment]
This should now be released as part of `v0.1.3`. Please test it and verify this is now fixed! Thank you!
